k8s/namespaces/private-law/prl-cos/ithc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
this file and prl-dgs don't appear to be referenced anywhere.

you're also using flux v2 for your identity and namespace which _may_ work but I _think_ generally you're recommended to use flux v1 outside of preview for now.